### PR TITLE
Fix inaccurate seeking.

### PR DESCRIPTION
**Pull request by hossein-zare**

### CODE REVIEW

**Review by freeboub — COMMENTED** [review]
Same patch available here: https://github.com/react-native-video/react-native-video/pull/2545

**Comment by volpstar** [comment]
@freeboub The linked patch was closed and not merged. Can you please re-review and merge if this PR looks good to you?

**Comment by hossein-zare** [comment]
@freeboub @volpstar 
I tested this PR (months ago) on Android 9 it worked well but on Android 5 the app crashed.

**Comment by freeboub** [comment]
@hossein-zare can you check if android api > 21 to enable the fix please ? @volpstar i don't have rights to merge PRs :)

**Review by freeboub — CHANGES_REQUESTED** [review]
Can you check api > 21 to use the second parameter, and remove commented line

**Comment by hossein-zare** [comment]
@freeboub I'm sorry, I don't work with react-native anymore.

**Comment by hueniverse** [comment]
We need two reviews approving this change to merge.

**Comment by SmartArray** [comment]
tACK.

We need this feature upstream please.

File: android/src/main/java/com/brentvatne/react/ReactVideoView.java [hunk 1]
```diff
@@ -637,7 +637,9 @@ public void onSeekComplete(MediaPlayer mp) {
     public void seekTo(int msec) {
         if (mMediaPlayerValid) {
             mSeekTime = msec;
-            super.seekTo(msec);
+            // super.seekTo(msec);
```
Comment (SmartArray) [inline android/src/main/java/com/brentvatne/react/ReactVideoView.java:640]: Remove this comment

**Review by SmartArray — CHANGES_REQUESTED** [review]
tACK

We need this feature upstream please.

**Comment by armands-malejevs** [comment]
@hueniverse I think this can be closed. This change is meant for the media player that has been removed in 6.0

**Comment by SmartArray** [comment]
@armadilio3 I am not quite sure, we are not using the media player but the seek function.
And the seek function works quite well on ios but not on android. I think this is still a bug that needs to be fixed?

**Comment by hueniverse** [comment]
@freeboub Can you help sort this out?

**Comment by freeboub** [comment]
I confirm this is old player, let's close this PR

**Pull request closed without merging**